library/common/http/header_utility.cc

```diff
@@ -0,0 +1,65 @@
+#include "library/common/http/header_utility.h"
+
+#include "common/http/header_map_impl.h"
+
+namespace Envoy {
+namespace Http {
+namespace Utility {
+
+static inline envoy_data copyEnvoyData(size_t length, const uint8_t* source) {
+  uint8_t* destination = static_cast<uint8_t*>(malloc(sizeof(uint8_t) * length));
+  memcpy(destination, source, length);
+  return {length, destination};
+}
+/*
```

Comment:
?

Reply:
Defined the function but not used yet anywhere in this code. Will delete for now.